speakers.js

```diff
@@ -0,0 +1,138 @@
+  },
+  {
+    id: 'speaker5',
+    image: 'https://images.weserv.nl/?url=emamo.com/storage/avatars/16/09/179285_7d272ed485eb9f82b314eaf835201609.jpg&w=400&h=400',
+    names: 'John Wolff',
+    company: 'Run it Wild Founder Inc',
+    description:
+      "Lorem Ipsum is simply dummy company of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy company ever since the 1500s.",
+  },
+  {
+    id: 'speaker6',
+    image: 'https://images.weserv.nl/?url=emamo.com/storage/avatars/aa/ba/179746_458d354d5149641fa54a0a3f0235aaba.jpg&w=400&h=400',
+    names: 'Aaron Albano',
+    company: 'President, Promzzy koncepts Inc',
+    description:
+      "Lorem Ipsum is simply dummy company of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy company ever since the 1500s.",
+  },
+  {
+    id: 'speaker2',
+    image: 'https://images.weserv.nl/?url=emamo.com/storage/avatars/5d/71/179495_90565705f8014d566989bf0b05d95d71.jpg&w=400&h=400',
+    names: 'Abel Czupor',
+    company: 'RadioShzck',
+    description:
+      "Lorem Ipsum is simply dummy company of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy company ever since the 1500s.",
+  },
+  {
+    id: 'speaker3',
+    image: 'https://images.weserv.nl/?url=emamo.com/storage/avatars/6e/e1/179467_6a90eeb8035ffe76e468d859c8506ee1.jpg&w=400&h=400',
+    names: 'Aaron Lane',
+    company: 'MINGs Music Enterprises LLC',
+    description:
+      "Lorem Ipsum is simply dummy company of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy company ever since the 1500s.",
+  },
+  {
+    id: 'speaker4',
+    image: 'https://images.weserv.nl/?url=emamo.com/storage/avatars/52/e5/180296_7f553e459f800d4de35b1d841d8e52e5.jpg&w=400&h=400',
+    names: 'Adam Amin',
+    company: 'CEO & Founder, Affectiva',
+    description:
+      "Lorem Ipsum is simply dummy company of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy company ever since the 1500s.",
+  },
+];
+
+let workCard = '';
+let workCard1 = '';
+const appendWorks = (speekerz, startt) => {
+  for (let i = startt; i < startt + 6; i += 1) {
+    workCard += ` <section class="speeker-card">
+    <img class="spk-img" src="${speekerz[i].image}" alt="${speekerz[i].names}" />
+    <div class="infor"><h3 class="namez">${speekerz[i].names}</h3>
+    <p class="card-co">
+    ${speekerz[i].company}
+    </p>
+    <p class="card-description">
+    ${speekerz[i].description}
+    </p></div> 
+  
+    </section>  `;
+  }
+  // if (window.innerWidth < 768) {
+  workCard += ' <div class="more"></div><button type="button"  class="btn buttonM" onclick="appendworksR(speekerz,5);"> <p>More &nbsp;<span class="downA">V</span></p> </button>';
```

Comment:
- [ ] In the mobile version, the more ⬇️  button does not work. The console error is `appendworksR is not defined`. Kindly fix the `onclick` handler to make it function correctly. 
Cheers and happy coding 👍 

![Capstone Project](https://user-images.githubusercontent.com/60618877/183243218-0f44989c-5758-4040-88d9-c341cff493d8.gif)